### ISSUE TITLE
[BUG]Missing language declaration

### ISSUE THREAD

**Issue by YuriDevAT** [YurisCodingClub/html-tag-quiz#8]
### Description

As a screen reader user, I want to listen to the correct pronunciation of the apps content. Each index.html should have set a language for its content in the corresponding language inside the html tag.

**Acceptance Criteria (AC)**
- [ ] a language attribute is set to english in the html tag

### Additional information

Useful link: https://www.w3schools.com/tags/att_lang.asp

**Comment by saffi-786** [comment]
What exactly are you looking to fix here?

**Comment by YuriDevAT** [comment]
There is no defined language for the website. Setting a lang attribute is crucial for screen reader users, or read-text-alound integrations.

**Comment by imbeshat** [comment]
Can I work on this if it's still open?

**Comment by CorinaMurg** [comment]
If not yet assigned, I would love to work on it! (And then we are done with the html bugs! 🎉)
Thank you so much!

**Comment by YuriDevAT** [comment]
Thanks for your interest, everyone. A real beginner touching GitHub for the first time will soon work on this. It's the perfect beginner issue to get into open source 🙇🏽

**Comment by Mr-Unforgettable** [comment]
Hey can I give it a shot. It will be a good starting point for me.

**Comment by Dorstener** [comment]
The beginner is in the house! Ready to go for it and thanks for saving this one up for me

**Comment by Dorstener** [comment]
Hi all,

Can I get a little bit of help here (sorry, I am the ultimate newbie). 

I have worked on this issue and added the missing language declaration to the html (_see _screenshot).

<img width="278" alt="image" src="https://github.com/YurisCodingClub/html-tag-quiz/assets/115483792/d3dd9071-067e-49bb-aace-d04ccb3a3313">

After that, I wanted to commit that, so:

1) I made sure I was on the branch i created for this issue:

<img width="332" alt="image" src="https://github.com/YurisCodingClub/html-tag-quiz/assets/115483792/9571d6c4-315e-4dd6-9a14-d0185beb7cb0">

2)  I used the following command (git add . ) 

<img width="334" alt="image" src="https://github.com/YurisCodingClub/html-tag-quiz/assets/115483792/7ca493ca-9cc5-4f33-b153-3fe5517ebbcb">

3) and  when I want to commit it but for some strange reason, it is not letting me execute the commit (see screenshot)

<img width="340" alt="image" src="https://github.com/YurisCodingClub/html-tag-quiz/assets/115483792/09fe16bd-1937-4b92-801d-9181aee89c28">

**NOTE:** 

a) I understand that the changes are not being added to commit. I also tried with "git add html-tag-quiz" since it is the modified content, but same

<img width="341" alt="image" src="https://github.com/YurisCodingClub/html-tag-quiz/assets/115483792/c48bc291-ac81-47db-8d94-2cd4df5c789b">

b) And I also tried with "git add -A + git commit -a" and nothing

<img width="350" alt="image" src="https://github.com/YurisCodingClub/html-tag-quiz/assets/115483792/4c2aee67-2fb6-47bb-a4bb-2b20c7544a34">

-----

What am I doing wrong? Thanks for the help and sorry for reaching out with such a basic thing

**Comment by CorinaMurg** [comment]
Hi @Dorstener! I would suggest to do a `git status`. This will show you any changes that are staged (and ready to be committed) and any that are not. Let's see that it says and then we can take it from there. No worries, it's always like that when starting with git.

**Comment by Dorstener** [comment]
> Hi @Dorstener! I would suggest to do a `git status`. This will show you any changes that are staged (and ready to be committed) and any that are not. Let's see what it says and then we can take it from there. No worries, it's always like that when starting with git!

Hi Corina, hereby the screenshot. 

<img width="374" alt="image" src="https://github.com/YurisCodingClub/html-tag-quiz/assets/115483792/08414d46-34ac-414c-a2c8-8bce0fec7716">


It takes me back to square one because the steps it suggest, I have done them already

**Comment by YuriDevAT** [comment]
If you are using Visual Studio Code, you can also work directly in the source control. Does the `index.html` file appear there? If yes, under `Changes`? Or under `Staged Changes`?
### Changes
<img width="285" alt="Screenshot 2023-10-26 at 13 32 38" src="https://github.com/YurisCodingClub/html-tag-quiz/assets/54622834/c7accc81-4494-4c2e-98b3-c9a28b59b2b7">
### Stages Changes
<img width="285" alt="Screenshot 2023-10-26 at 13 34 03" src="https://github.com/YurisCodingClub/html-tag-quiz/assets/54622834/38d28ec9-b970-4eb8-a4f1-c3709bccd35e">

**Comment by CorinaMurg** [comment]
Good idea @YuriDevAT! 
(I noticed that git status shows `modified: html-tag-quiz`.  I would expect to see `modified: index.html`. We can look into that if committing directly from VSCode does not work)

**Comment by Dorstener** [comment]
This is how it is showing from the source control

<img width="259" alt="image" src="https://github.com/YurisCodingClub/html-tag-quiz/assets/115483792/ea49e1f8-1f44-4761-9f00-a45092900955">

And yes, I believe the branch is created from the master and not from the fork

<img width="327" alt="image" src="https://github.com/YurisCodingClub/html-tag-quiz/assets/115483792/d53ba7b6-8630-4f52-8f1a-d0abd6d2dade">

**Comment by CorinaMurg** [comment]
I noticed that you are working in a folder called `opensource`, but git status says the modified file is `html-tag-quiz`, so I am suspecting a nested folder issue. In this scenario, you opened the `opensource` folder in VSCode and worked on the `html-tag-quiz` folder inside it. But when you used `git add .`,  it looked for a Git repository called `opensource` but didn't find one. 

If this is the case and you do have the forked repo inside the opensource folder, try this: undo the change and close the opensource folder. Then, in VSCode when you click File - Open Folder go to opensource, then select the html-tag-quiz folder to open it. In VSCode you should see:
![image](https://github.com/YurisCodingClub/html-tag-quiz/assets/115652409/a5edf6e5-061e-4a47-8716-647f5795e0b1)

**Comment by Dorstener** [comment]
Hi Corina, 

I believe it is done and the pull request sent.

As you mentioned, I believe it was the fact that I was mixing root folders, between opensource and html-tag-quiz.

I have submitted the pull request, let's hope it works. 

Thanks a lot for the help :)

**Comment by CorinaMurg** [comment]
@Dorstener 
Glad it worked!
